lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_model_base.rb

```diff
@@ -114,7 +122,7 @@ def self.create_service_model(ar_model)
     def self.dynamic_service_model_creation(ar_model, super_class)
       Class.new(super_class) do |klass|
         ::MiqAeMethodService.const_set(model_to_service_model_name(ar_model), klass)
-        expose_class_attributes(klass)
+        inherited(klass)
```

Comment:
This is scary.  Sorry, I didn't notice this before.

Can we move this expose logic from inherited to a new method `expose_the_stuff` and call it here and from inherited?